### DISCUSSION TITLE
elevator pitch

### DISCUSSION THREAD

**Discussion by harpomaxx** [harpomaxx/algoritmos2#65] [General]
elevator pitch: (tienen 1min para explicar cual es la diff entre la LCC y otras carreras relacionadas con la informatica) alguna idea?

**Comment by adriano-229** [comment]
"La diferencia principal es que la Licenciatura (CS Degree) se enfoca más en los aspectos teóricos, matemáticos y ontológicos de la computación, en comparación con las (otras) carreras a fines, que tienden a orientarse más a la implementación práctica de estos aspectos. Sin embargo, generalmente la salida laboral suele ser la misma."

Eso es lo que suelo responder, no sé si está equivocado o es poco riguroso, pero sirve jaja.

**Reply by harpomaxx** [reply]
suena bien... me gusto lo de ontologicos ja ja. ahora pensa que la idea es  explicárselo a gente que no tiene conocimientos de informática... necesariamente.

**Comment by JulianIgnFer24** [comment]
"La diferencia es que la licenciatura se basa en resolver problemas (de cualquier tipo mientras se puedan aplicar los conocimientos científicos y conocimientos de Informática) de manera eficiente, ya que la carrera tiene una base matemática bastante fuerte. Por lo tanto, se tiende a mirar un problema de forma más abstracta. Como dice mi compañero: 'se enfoca en aspectos más teóricos, mientras que otras carreras se enfocan más en lo práctico"

**Reply by harpomaxx** [reply]
si, tambien es una respuesta valida.. aunque uno podría argumentar que en otras carreras también se resuelven problemas de cualquier índole...
lo de una base teorica va muy bien!!

Gracias por responder!!

**Comment by Luci435** [comment]
La Lic se centra mucho mas en los aspectos teóricos, matemáticos e incluso abstractos, se dedica mas tiempo a la programación y no se relacion tanto con lo empresarial y las organizaciones. Además en otras carreras se centran mas en el hardware y las redes cuando en la Lic se ve pero no es lo que define la carrera.

**Reply by harpomaxx** [reply]
Es valido tambien, de alguna manera es mucho de lo que yo por ahí decía para vender la carrera. Aspectos teoricos, mas matematica, mucha programación.. En relación por ej. a la Ing o Lic en informática de la UM donde se programa mucho también, pero se ve mucho más de redes.  

Por otro lado, hoy con los algoritmos y técnicas de IA para generacion de codigo, muchos alegan que la programación ya no será importante.. esta entonces condenada una carrera como la nuestra?

Uds que opinan?

BTW, gracias por participar Lucia..

**Reply by Luci435** [reply]
Para mi no lo está, principalmente porque no está tan avanzada la IA como para suplantarnos y asumiendo que va a avanzar y mejorar, se necesita minimamente alguien que programe esa IA y que la mejore por ende programadores se van a seguir necesitando. Además uno no puede simplemente pedir un codigo y que este sea perfecto, hay que saber qué pedir, como hacerlo y como usarlo de base para complementar. Es mas un trabajo en conjunto que uno por el otro creo yo.
